03-plugins.md

```diff
@@ -55,9 +54,9 @@ Adding interaction to the Toolbar.
 $(document).ready(function() {
   $('#summernote').summernote({
     toolbar:[
-      // This is a Custom Button in a new Toolbar Area
+// This is a Custom Button in a new Toolbar Area
```

Comment:
Is removing blanks(indents) better for readability?

Reply:
I thought so, just to make things easier to read.

Reply:
I thought it is better to include indentation than removing for readability.

Reply:
I fixed the indentation so it's the same as other areas of the Website.